### PR TITLE
Introduce ENABLE_AMAVIS env

### LINKED ISSUE #465: Add ENABLE_AMAVIS env
Follow-up of #386

### PR DESCRIPTION
**Pull request by casperklein**
# Description

This allows to disable Amavis. This is useful, when ClamAV and Spamassassin are turned off.

Fixes #465

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change that does improve existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (README.md or ENVIRONMENT.md or the Wiki)
- [ ] If necessary I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by fbartels — DISMISSED** [review]

**Review by aendeavor — DISMISSED** [review]

**Comment by aendeavor** [comment]
I'd leave it up to @wernerfred or @polarathene to merge this, as I do not know whether this may interfere with #1826. Currently, only rebasing onto current master would need to be done, but really I'm not sure.

The changes look good though!

**Comment by wernerfred** [comment]
We will Take Care of it (probably at Weekend when Wiki pr gets merged)

File: ENVIRONMENT.md [hunk 1]
```diff
@@ -27,6 +27,13 @@ Here you can adjust the [log-level for Supervisor](http://supervisord.org/loggin
 
 The log-level will show everything in its class and above.
 
+##### ENABLE_AMAVIS
+
+Amavis content filter (used for ClamAV & Spamassassin)
```
Comment (williamdes) [inline ENVIRONMENT.md:32]:
```suggestion
Amavis content filter (used for ClamAV & SpamAssassin)
```

File: mailserver.env [hunk 2]
```diff
@@ -64,6 +64,11 @@ ENABLE_SRS=0
 ENABLE_POP3=
 ENABLE_CLAMAV=0
 
+# Amavis content filter (used for ClamAV & Spamassassin)
```
Comment (williamdes) [inline mailserver.env:67]:
```suggestion
# Amavis content filter (used for ClamAV & SpamAssassin)
```

**Review by williamdes — DISMISSED** [review]

**Review by aendeavor — APPROVED** [review]

**Review by wernerfred — APPROVED** [review]

**Comment by georglauterbach** [comment]
@all-contributors please add @casperklein for maintenance

**Pull request merged**